tests/functional/test_install.py

```diff
@@ -5,6 +5,7 @@
 import textwrap
 from os.path import curdir, join, pardir
 
+import pip._vendor.six  # noqa: F401
```

Comment:
Have you tried seeing if the tests work with the import statement removed? Since "pip._vendor.six.PY3" occurs inside a string, the import might be happening dynamically on its own.

Reply:
Without the import:
```
Error evaluating 'skipif' expression
    pip._vendor.six.PY2
NameError: name 'pip' is not defined
```
https://travis-ci.org/hugovk/pip/jobs/563611328#L326

I've added the comment in https://github.com/pypa/pip/pull/6782/commits/b4ac45cdd3db50f3cb1271853a40deb1a53d0703.